.env

```diff
@@ -120,3 +120,8 @@ OPENMRS_CONFIG_CHECKSUMS_PATH=
 MYSQL_DATADIR=
 POSTGRES_DATADIR=
 ODOO_FILESTORE=
+
+#
+# Frontend tag
+#
+TAG=3.0.0-beta.2
```

Comment:
Why's that, where's this used? Looks like this is not a good naming anyway, it's too vague. If we need to keep it it should become something like `O3_FRONTEND_TAG` no?

Reply:
I have removed it

Reply:
I've demonstrated it here https://github.com/ozone-his/ozone-docker/pull/30. Could be merged if okay.